the-experts-fe/src/App.js

```diff
@@ -3,16 +3,32 @@ import logo from "./logo.svg";
 import "./App.css";
 
 class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { apiResponse: "" };
+  }
+
+  callAPI() {
+    fetch("http://localhost:9000/testAPI")
```

Comment:
instead of adding the url manually you should have a `.env` file in the react app, [check this out ](https://create-react-app.dev/docs/adding-custom-environment-variables/#adding-development-environment-variables-in-env)